src/Type/BasePickerType.php

```diff
@@ -48,10 +47,9 @@ abstract class BasePickerType extends AbstractType
 
     public function __construct(MomentFormatConverter $formatConverter, $translator, ?RequestStack $requestStack = null)
     {
-        if (!$translator instanceof LegacyTranslatorInterface && !$translator instanceof TranslatorInterface) {
+        if (!$translator instanceof TranslatorInterface) {
             throw new \InvalidArgumentException(sprintf(
-                'Argument 2 should be an instance of %s or %s',
-                LegacyTranslatorInterface::class,
+                'Argument 2 should be an instance of %s',
                 TranslatorInterface::class
             ));
         }
```

Comment:
Maybe we can use a type declaration instead now? It's not a BC-break, is it?

Reply:
Well, if you are extending this class, you could have this without type hint, even if you are always passing the same argument

Reply:
I think it could be safer this way, but I can change it if needed

Reply:
The LSP does not apply to constructors, if that's why you are talking about extending classes. And anyway, not have the type hint is no longer an issue since php 7.2, which this package requires.

Reply:
Changed